ui/components/src/LineChart/LineChart.tsx

```diff
@@ -89,26 +80,7 @@ export interface LineChartProps {
   __experimentalEChartsOptionsOverride?: (options: EChartsCoreOption) => EChartsCoreOption;
 }
 
-// Intentionally making this an object to start because it is plausible we will
-// want to support focusing by other attributes (e.g. index, name) in the future,
-// and starting with an object will make adding them a non-breaking change.
-type LineChartHandleFocusOpts = {
-  id: string;
-};
-
-export type LineChartHandle = {
```

Comment:
What is "handle" meant to mean in this situation? It looks like this object is related to highlighting series, but I can't quite figure out what that has to do with a "handle".

Reply:
Yeah that came in with recent legend changes, just renamed to ChartInstance